extraction/embedded_tweet_extractor.py

```diff
@@ -0,0 +1,22 @@
+"""This script checks whether DOM has embedded tweet tag or not and
+creates and returns the ETweet object"""
+
+from extraction.data_models.embedded_tweet import ETweet
+from extraction.interface_content_extractor import IContentExtractor
+
+
+class ETweetExtractor(IContentExtractor):
+    """This class inherits IContentExtractor for extracting embedded tweets"""
+
+    def validate_and_extract(self, node):
+        try:
+            if node['class'] == ['twitter-tweet', 'twitter-tweet-rendered'] \
```

Comment:
Do we need try/catch if regex is well defined?

Reply:
Try/catch is there for cases when the any attribute or key is not found. We can use if/else too. Any suggestions?

Reply:
Added if conditions as we don't need try/catch specifically here.